### ISSUE TITLE
Redirects to do during PI14.1

### ISSUE THREAD

**Issue by kathycarothers** [fecgov/fec-cms#4423]
### Summary

**What we're after:**
_We are getting off of transition. Old files need to be redirected to their new homes before we can take 'em down._

### Related issues

_List any relevant related issue(s)_

- #4380 
- Accessibility redirects - see spreadsheet - www - these have been moved to fecgov/fec-proxy#240
- Audit document Erick Paulsen redirect added (this is a www redirect) 

### Completion criteria

- [x] Redirects are merged into the release
- [x] 2021 spreadsheet is updated
- [x] Testing ticket is created to be done in the next sprint after deploy of releases. Also will need to check www redirect after release deploy. - https://github.com/fecgov/fec-cms/issues/4433

### Future work

_Once redirects have been deployed need to alert frontend to delete these transition files in the folder. See ticket #4380_

**Comment by kathycarothers** [comment]
The accessibility links that were listed were already entered. I have put in the redirects for the Enforcement links from the inventory list of items to remove off transition.  Once they have been reviewed I will update this ticket.

**Comment by dorothyyeager** [comment]
Testing ticket for all 14.1 redirects created at https://github.com/fecgov/fec-cms/issues/4433.
